thoth/python/package_version.py

```diff
@@ -34,6 +33,10 @@
 from .exceptions import InternalError
 from .source import Source
 
+from thoth.python.pipfile import PipfileMeta
```

Comment:
This is causing the whole issue of PackageVersion cant be imported. 
also this is not used anywhere in here, please remove this.

Reply:
Harshad, I think it's being used, for example in this line: https://github.com/thoth-station/python/blob/master/thoth/python/package_version.py#L184, but there's a spelling error, it's written as `PipenvMeta` and it needs to be `PipfileMeta`.

Reply:
If we remove that, then `pre-commit` won't be happy:
```
thoth/python/package_version.py:188: error: Name 'PipfileMeta' is not defined
thoth/python/package_version.py:208: error: Name 'PipfileMeta' is not defined
thoth/python/package_version.py:293: error: Name 'PipfileMeta' is not defined
Found 3 errors in 1 file (checked 12 source files)

black....................................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

thoth/python/package_version.py:188:36: F821 undefined name 'PipfileMeta'
thoth/python/package_version.py:208:91: F821 undefined name 'PipfileMeta'
thoth/python/package_version.py:293:98: F821 undefined name 'PipfileMeta'
```

Reply:
ah, if it's being used then this going into looping import issue, we would have to rewrite and move around components.
Also, why is the PipfileMeta in quotes in line https://github.com/thoth-station/python/blob/master/thoth/python/package_version.py#L184 ?

Reply:
Harshad, I have no idea, i think it works either way. First when we explicitly import the class and its name and maybe second way is the current way where we write it in quotes, but not sure.